### PR TITLE
victorialogs quickstart

### PR DESCRIPTION
**Pull request by Amper**

### CODE REVIEW

File: docs/VictoriaLogs/README.md [hunk 1]
```diff
@@ -23,263 +23,18 @@ VictoriaLogs is at Preview stage now. It is recommended evaluating it on product
...
-```
-
-VictoriaLogs stores the ingested data to the `victoria-logs-data` directory by default. The directory can be changed
-via `-storageDataPath` command-line flag. See [these docs](#storage) for details.
-
-By default VictoriaLogs stores log entries with timestamps in the time range `[now-7d, now]`, while dropping logs outside the given time range.
-E.g. it uses the retention of 7 days. Read [these docs](#retention) on how to control the retention for the [ingested](#data-ingestion) logs.
-
-It is recommended setting up monitoring of VictoriaLogs according to [these docs](#monitoring).
+See [Quick Start](https://docs.victoriametrics.com/VictoriaLogs/quickstart.html) for getting started with VictoriaLogs.
```
Comment (tenmozes) [inline docs/VictoriaLogs/README.md:28]: Could you please define the separate section and Call it Quickstart - we definitely will want to provide the anchored link to it

File: docs/VictoriaLogs/README.md [hunk 2]
```diff
@@ -23,263 +23,18 @@ VictoriaLogs is at Preview stage now. It is recommended evaluating it on product
...
-    }
-    parameters => {
-        "_msg_field" => "message"
-        "_time_field" => "@timestamp"
-        "_stream_fields" => "host.hostname,process.name"
-    }
-  }
-}
-```
+VictoriaLogs can accept logs from popular log collectors via following data ingestion protocols:
```
Comment (tenmozes) [inline docs/VictoriaLogs/README.md:32]: Please list the protocols here and provide links to collectors.md to each type of collector

File: docs/VictoriaLogs/Roadmap.md [hunk 3]
```diff
@@ -6,18 +6,16 @@ It isn't recommended migrating from existing solutions to VictoriaLogs Preview y
 
 The following functionality is available in VictoriaLogs Preview:
 
-- [Data ingestion](https://docs.victoriametrics.com/VictoriaLogs/#data-ingestion).
+- [Data ingestion](https://docs.victoriametrics.com/VictoriaLogs/ingestion.html).
 - [Querying](https://docs.victoriametrics.com/VictoriaLogs/#querying).
 - [Querying via command-line](https://docs.victoriametrics.com/VictoriaLogs/#querying-via-command-line).
 
-See [operation docs](https://docs.victoriametrics.com/VictoriaLogs/#operation) for details.
+See [operation docs](https://docs.victoriametrics.com/VictoriaLogs/quickstart.html) for details.
```
Comment (tenmozes) [inline docs/VictoriaLogs/Roadmap.md:13]: should we change the title from operation docs to quickstart? just not to confuse people but still mention the purpose

File: docs/VictoriaLogs/collectors.md [hunk 4]
```diff
@@ -0,0 +1,554 @@
...
+
+Substitute the `localhost:9428` address inside `hosts` section with the real TCP address of VictoriaLogs.
+
+The `_msg_field` parameter must contain the field name with the log message generated by Filebeat. This is usually `message` field.
+See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#message-field) for details.
+
+The `_time_field` parameter must contain the field name with the log timestamp generated by Filebeat. This is usually `@timestamp` field.
+See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#time-field) for details.
+
+It is recommended specifying comma-separated list of field names, which uniquely identify every log stream collected by Filebeat, in the `_stream_fields` parameter.
```
Comment (tenmozes) [inline docs/VictoriaLogs/collectors.md:29]: a bit vise-versa

We recommend leaving it empty unless the user exactly knows the good streaming fields. We can provide examples

File: docs/VictoriaLogs/collectors.md [hunk 5]
```diff
@@ -0,0 +1,554 @@
...
+
+Substitute the address (`localhost`) and port (`9428`) inside `Output` section with the real TCP address of VictoriaLogs.
+
+The `_msg_field` parameter must contain the field name with the log message generated by Fluentbit. This is usually `message` field.
+See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#message-field) for details.
+
+The `_time_field` parameter must contain the field name with the log timestamp generated by Fluentbit. This is usually `@timestamp` field.
+See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#time-field) for details.
+
+It is recommended specifying comma-separated list of field names, which uniquely identify every log stream collected by Fluentbit, in the `_stream_fields` parameter.
```
Comment (tenmozes) [inline docs/VictoriaLogs/collectors.md:129]: as above

File: docs/VictoriaLogs/collectors.md [hunk 6]
```diff
@@ -0,0 +1,554 @@
...
+
+Substitute `localhost:9428` address inside `hosts` with the real TCP address of VictoriaLogs.
+
+The `_msg_field` parameter must contain the field name with the log message generated by Logstash. This is usually `message` field.
+See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#message-field) for details.
+
+The `_time_field` parameter must contain the field name with the log timestamp generated by Logstash. This is usually `@timestamp` field.
+See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#time-field) for details.
+
+It is recommended specifying comma-separated list of field names, which uniquely identify every log stream collected by Logstash, in the `_stream_fields` parameter.
```
Comment (tenmozes) [inline docs/VictoriaLogs/collectors.md:204]: as above

File: docs/VictoriaLogs/ingestion.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+{ "log": { "level": "info", "message": "hello world" }, "date": "2023‐06‐20T15:35:11Z", "stream": "stream2" }
+```
+
+Here are extra query parameters, which can be passed to the `/insert/jsonline` endpoint:
+
+- The `_msg_field` parameter must contain the field name with the log message generated by collector. This is usually `message` field.
+  See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#message-field) for details.
+- The `_time_field` parameter must contain the field name with the log timestamp generated by collector. This is usually `@timestamp` field.
+  See [these docs](https://docs.victoriametrics.com/VictoriaLogs/keyConcepts.html#time-field) for details.
+- It is recommended specifying comma-separated list of field names, which uniquely identify every log stream collected by Filebeat, in the `_stream_fields` parameter.
```
Comment (tenmozes) [inline docs/VictoriaLogs/ingestion.md:86]: as above

File: docs/VictoriaLogs/quickstart.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+VictoriaLogs automatically adapts to the available CPU and RAM resources. It also automatically setups and creates
+the needed indexes during [data ingestion](https://docs.victoriametrics.com/VictoriaLogs/ingestion.html).
+
+## How to configure VictoriaLogs
+
+It is possible to change the TCP port via `-httpListenAddr` command-line flag. For example, the following command
+starts VictoriaLogs, which accepts incoming requests at port `9200` (aka ElasticSearch HTTP API port):
+
+```bash
+/path/to/victoria-logs -httpListenAddr=:9200
```
Comment (tenmozes) [inline docs/VictoriaLogs/quickstart.md:53]: let's specify the default configuration and add a line how people can customize to mimic Elastic

**Comment by valyala** [comment]
@Amper , thanks for the docs re-haul! I couldn't merge the pull request as is because of conflicts with the updated docs, so I manually applied the changes from this pull request to 332b29526800c3ec8f05e33070ba292ddcde990b . This commit reflects all the changes from the pull request except of docs for data ingestion with JSON line protocol. These docs will be included after merging https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4469

**Pull request closed without merging**